antora-ui-camel/src/css/header.css

```diff
@@ -130,6 +130,8 @@ body {
 .navbar-item {
   flex: none;
   margin-bottom: 1px;
+  font-weight: bolder;
```

Comment:
this made menu items in burger menu more visible

Reply:
This is overridden here:

https://github.com/apache/camel-website/blob/fb0ff7bbfe19b2b98d7be22d158a3676c55dc85d/antora-ui-camel/src/css/nav.css#L187

So it doesn't have an effect I'm afraid. Also would be good to use variables so we can tweak from vars.css.